pandasql/core.py

```diff
@@ -398,6 +398,26 @@ def __setitem__(self, col, value):
         # Update self to new object
         self.__dict__.update(new.__dict__)
 
+    def rename(self, columns):
```

Comment:
So we don't support in-place renames, correct?

Reply:
yes, in place is not supported